bust-a-move-framework/src/test/java/bustamove/game/CannonTest.java

```diff
@@ -43,8 +44,9 @@ public void fireMaxRighttAngle() {
         for (int i = 0; i < 60; i++) {
             cannon.stepUp();
         }
+        Bubble loaded = game.getBubbles().get(1);
```

Comment:
Originally the bubble that was most recently shot would be in the third position on the bubbleslist, Now the shot bubble will always be in the 2nd. 

The order of the bubbles is a bit strange tho..